hack/lib/etcd.sh

```diff
@@ -18,9 +18,14 @@
 # A set of helpers for starting/running etcd for tests
 
 ETCD_VERSION=${ETCD_VERSION:-3.3.10}
-INT_NAME="eth0"
+INT_NAME=${INT_NAME:-eth0}
```

Comment:
This will still default to eth0 if not specified. Please add code to query the default interface using ip route.

Reply:
Updated it to ip route | awk '/default/ { print $5 }'